public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_resource_graph/azure_resource_graph_datasource.ts

```diff
@@ -53,4 +55,41 @@ export default class AzureResourceGraphDatasource extends DataSourceWithBackend<
     });
     return quotedValues.join(',');
   }
+
+  query(request: DataQueryRequest<AzureMonitorQuery>): Observable<DataQueryResponse> {
+    const metricQueries = request.targets.reduce((prev: Record<string, AzureMonitorQuery>, cur) => {
+      prev[cur.refId] = cur;
+      return prev;
+    }, {});
+
+    return super.query(request).pipe(
+      mergeMap((res: DataQueryResponse) => {
+        return from(this.processResponse(res, metricQueries));
+      })
+    );
+  }
+
+  async processResponse(
+    res: DataQueryResponse,
+    metricQueries: Record<string, AzureMonitorQuery>
+  ): Promise<DataQueryResponse> {
+    if (res.data) {
+      for (const df of res.data) {
+        const metricQuery = metricQueries[df.refId];
+        if (metricQuery && metricQuery.azureMonitor) {
+          const url = 'https://portal.azure.com/#blade/HubsExtension/ArgQueryBlade';
```

Comment:
National clouds which have different portal address

Reply:
I see metric and log have the same issue. Maybe we could make this a separate fix?